x-pack/solutions/observability/plugins/apm/server/routes/correlations/queries/fetch_duration_histogram_range_steps.ts

```diff
@@ -69,24 +71,28 @@ export const fetchDurationHistogramRangeSteps = async ({
       }
     : query;
 
-  const resp = await apmEventClient.search('get_duration_histogram_range_steps', {
-    apm: {
-      events: [getEventType(chartType, searchMetrics)],
+  const resp = await apmEventClient.search(
+    'get_duration_histogram_range_steps',
+    {
+      apm: {
+        events: [getEventType(chartType, searchMetrics)],
+      },
+      track_total_hits: 1,
+      size: 0,
+      query: getCommonCorrelationsQuery({
+        start,
+        end,
+        environment,
+        kuery,
+        query: filteredQuery,
+      }),
+      aggs: {
+        duration_min: { min: { field: durationField } },
+        duration_max: { max: { field: durationField } },
+      },
     },
-    track_total_hits: 1,
-    size: 0,
-    query: getCommonCorrelationsQuery({
-      start,
-      end,
-      environment,
-      kuery,
-      query: filteredQuery,
-    }),
-    aggs: {
-      duration_min: { min: { field: durationField } },
-      duration_max: { max: { field: durationField } },
-    },
-  });
+    { skipProcessorEventFilter: isOtel }
```

Comment:
These APM queries assume we will be filtering by the `processor.event` field, but for unprocessed OTEL, we can just skip this